cmd/cluster-server/servergrpc/cluster-server.proto

```diff
@@ -0,0 +1,128 @@
+syntax = "proto3";
+
+package servergrpc;
+
+import "google/api/annotations.proto";
+
+service ClusterServerService {
+        rpc GetClientStream (stream ClientMes) returns (stream ClientMes) {}
```

Comment:
yes sure, but first get agreement on the cli verbs/structure and inclusion or not in amp cli.